domain/src/main/java/com/globant/domain/repository/CharacterRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.globant.domain.repository
+
+import com.globant.domain.entity.Character
+import com.globant.domain.util.Result
+
+interface CharacterRepository {
+    fun getAllLocalCharacters(): Result<List<Character>>
+    fun getSpecificLocalCharacter(characterId: String): Result<List<Character>>
+    fun insertLocalCharacters(characters: List<Character>)
+}
```

Comment:
pls add the line at he end of the file

Reply:
check this is repeated in most of the files

Reply:
added the spaces on the files that we talk, thanks